helm/kiam/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kiam
-version: 4.0.0
+version: 4.1.0
```

Comment:
Sorry - #316 merged previously has bumped the version to `4.1.0` already, can we bump to `4.2.0` please